src/inc/quic_platform.h

```diff
@@ -147,9 +147,9 @@ DEFINE_ENUM_FLAG_OPERATORS(CXPLAT_THREAD_FLAGS);
 #elif _WIN32
 #define CX_PLATFORM_TYPE 2
 #include <quic_platform_winuser.h>
-#elif CX_PLATFORM_LINUX
+#elif CX_PLATFORM_LINUX || CX_PLATFORM_DARWIN
 #define CX_PLATFORM_TYPE 3
-#include <quic_platform_linux.h>
+#include <quic_platform_posix.h>
 #else
```

Comment:
What do you think about defining different `CX_PLATFORM_TYPE` values here instead, but both still including *posix.h?